paper-server/src/main/java/org/bukkit/craftbukkit/inventory/CraftContainer.java

```diff
@@ -123,7 +123,7 @@ public static net.minecraft.world.inventory.MenuType getNotchInventoryType(Inven
                     case 27:
                         return net.minecraft.world.inventory.MenuType.GENERIC_9x3;
                     case 36:
-                    case 41: // PLAYER
+                    case 43: // PLAYER
```

Comment:
Is there some constant or some calculation we can do from the nms.Inventory type to automatically have this value update?

Reply:
Also I could remove handling of player inventory, as inventory type already has the good menu type, so the default case would handle it perfectly fine

Reply:
That sounds smart :+1: